### PR TITLE
[REFACTOR]: update legacybox with new components

### PR DESCRIPTION
**Pull request by ChiragPansuriya-iView**
- [x] Replaced LegacySecondaryBox with Box in Pagination.tsx
- [x] Replaced LegacyPrimaryBox with Box in FileUploader.tsx and FileUploader.web.tsx
- [x] Replaced All LegacyTertiaryBox with TertiaryBox in Footer.tsx
- [x] Replaced LegacyPrimaryBox with Box in ToolbarContainer.tsx
- [x] Replaced LegacySecondaryBox with Box in Dropdown.tsx
- [x] Replaced LegacyTertiaryBox with TertiaryBox in LaunchpadButton.tsx

### CODE REVIEW

**Comment by zxxma** [comment]
thanks for this 👍

File: packages/components/socialFeed/RichText/Toolbar/ToolbarContainer.tsx [hunk 1]
```diff
@@ -6,26 +6,33 @@ import {
...
+      borderGradient={{
+        start: { x: 0, y: 0 },
+        end: { x: 1, y: 1 },
+        locations: [0.7, 0.8],
+        colors: [
+          gradientColorDarkerBlue,
+          gradientColorBlue,
+          gradientColorTurquoise,
+        ],
+      }}
```
Comment (n0izn0iz) [inline packages/components/socialFeed/RichText/Toolbar/ToolbarContainer.tsx:23]: there's a regression here
before:
Screenshot 2023-12-21 at 19 24 11
after:
Screenshot 2023-12-21 at 19 24 36

**Review by n0izn0iz — CHANGES_REQUESTED** [review]
one usage has a regression

File: packages/components/socialFeed/RichText/Toolbar/ToolbarContainer.tsx (hunk 1 above)
Comment (ChiragPansuriya-iView) [reply packages/components/socialFeed/RichText/Toolbar/ToolbarContainer.tsx:23]: @n0izn0iz Thanks for notifying, The issue is fixed now.

**Comment by omniwired** [comment]
@ChiragPansuriya-iView thanks for your contribution.
I feel that "notch" is a bit obscure of a word for non-advanced English speakers.
How about changing any Notch elements to IndentedX for example: IndentedBox
what do the rest of the team think? cc: @n0izn0iz @clegirar @WaDadidou @hthieu1110

**Review by omniwired — CHANGES_REQUESTED** [review]
@ChiragPansuriya-iView thanks for your contribution.
I feel that "notch" is a bit obscure of a word for non-advanced English speakers.
How about changing any Notch elements to IndentedX for example: IndentedBox

Same with notch style, etc.

what do the rest of the team think? cc: @n0izn0iz @clegirar @WaDadidou @hthieu1110

**Comment by n0izn0iz** [comment]
I chose the notch wording, that's what people on internet called "broken corners" boxes

**Comment by n0izn0iz** [comment]
I'm asking zooma (design) if he prefers to keep gradient or notch

**Comment by n0izn0iz** [comment]
ok let's keep the gradient but no notches, please revert the changes on Box

**Comment by ChiragPansuriya-iView** [comment]
> ok let's keep the gradient but no notches, please revert the changes on Box

> I have reverted box changes please review it
Screenshot 2023-12-23 at 9 47 29 AM

**Comment by clegirar** [comment]
Hey 👋

First of all thanks for your contribution, so nice to see first PR's that do real refracto !
Glad to have you as a contributor, don't hesitate to ask questions on discord whenever you want 👌

For `Boxes` imo, we don't have to use `Box` component directly in screens or high level components because we can easily have some regression in the future, it's so easy to put something bad in it (not intentionally) and reviewers can didn't see the new `style` prop for example
I think that `Box` component have to be "private" to specific boxes components (`PrimaryBox`, `SecondaryBox` and `TertiaryBox`, or maybe other ones in the future) and ONLY used by these components
It join globally this discussion https://discord.com/channels/876922049279455274/1177279804295348376/1177286731595255940
So imo we have to use one of `PrimaryBox`, `SecondaryBox` or `TertiaryBox` 👍 I don't know if it's the goal of the PR but if we want to do that correctly i think it's the way 👍 

Wdyt about it ? @n0izn0iz @ChiragPansuriya-iView

Else the code is good 👍 
Maybe you can refacto your commits (you have 2 commits with the same name) and i think that you didn't correctly rebased `main`
And for you commits names you can just put `fix:` as prefix, full lowercase and don't need brackets `[]`. Same for your PR's name 👍 

Thanks again and don't hesitate to ping us 😊

**Pull request closed without merging**